_posts/2017/2017-04-06-javaland-2017-in-phantasialand.md

```diff
@@ -0,0 +1,75 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# People, Fun and Food
+
+The conference is all about meeting great Java software engineers from other countries as well, including real [Java Champions](https://community.oracle.com/community/java/java-champions) and [JavaOne Rockstars](https://www.oracle.com/javaone/rock-star-wall-of-fame.html), and meeting Java User Groups from Germany (see [iJUG](ijug.eu), the umbrella organisation of many [JUG_DE](https://twitter.com/JUG_DE)) from all around the world.
+The team from [NightHacking](http://nighthacking.com) even had a very long live streams from the exhibition room, where famous companies offered support – like the [Eclipse Foundation](https://eclipse.org/org/foundation/) or [JetBrains](https://www.jetbrains.com/), which even came with core developers of the well-known [IDEA](https://www.jetbrains.com/idea/) programming environment for all JVM languages.
+It was also great to meet the founder and organizer of the biggest Java User Group on our planet, the [Virtual Java User Group](https://virtualjug.com/) by [Simon Maple](https://twitter.com/sjmaple) and [Oleg Shelajev](https://twitter.com/shelajev) from [ZeroTurnaround](https://zeroturnaround.com/) aka the really helpful [JRebel](https://zeroturnaround.com/software/jrebel) tool company.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/JAVA?src=hash">#JAVA</a> champions + <a href="https://twitter.com/hashtag/JCP?src=hash">#JCP</a> leaders <a href="https://twitter.com/JavaLandConf">@JavaLandConf</a> with <a href="https://twitter.com/shelajev">@shelajev</a> <a href="https://twitter.com/dataduke">@dataduke</a><a href="https://twitter.com/miragemiko">@miragemiko</a> <a href="https://twitter.com/sjmaple">@sjmaple</a> <a href="https://twitter.com/nighthacking">@nighthacking</a> <a href="https://twitter.com/hashtag/JavaLand?src=hash">#JavaLand</a> <a href="https://t.co/ISjvJrvYwU">pic.twitter.com/ISjvJrvYwU</a></p>&mdash; Benjamin Nothdurft (@dataduke) <a href="https://twitter.com/dataduke/status/846685232346615813">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# Summary
+
+Everything was awesome – from start to finish.
+I hope we see you next year there as well.
+And don't forget about all the free roller coaster rides without waiting time, the great late night music concert with a terrific live band and the free food at every Restaurant at the marvelous and epic [Phantasialand](http://www.phantasialand.de/en/) near Cologne, Germany.
```

Comment:
...with a terrific live band, and the free food at every restaurant at the marvelous and epic...